### PR TITLE
Feat/documentation

### PR DESCRIPTION
**Pull request by danifromecuador**
## In this PR, I've:
- Created a `readme.md` file in a professional way

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,220 @@
...
+Contributions are welcome! If you find any bugs or want to enhance the project with new features, please follow these steps:
+
+1. Fork the repository.
+2. Create a new branch for your feature or bug fix.
+3. Make the necessary changes and commit them.
+4. Push your changes to your forked repository.
+5. Submit a pull request describing your changes.
+
+Feel free to check the [issues page](../../issues/).
+
```
Comment (Diegogagan2587) [inline README.md]: Link to issue page should match the github issue page:
https://github.com/Diegogagan2587/Book-a-concert-back-end/issues

**Review by Diegogagan2587 — CHANGES_REQUESTED** [review]
# Status: Almost done :orange_circle: :orange_circle: :orange_circle: 

## Required changes:

please upadte the issues page to match the project issues page, mor info on the comments

File: README.md (hunk 1 above)
Comment (danifromecuador) [reply README.md]: Thank you for catching that detail, I fixed that and other broken links

**Review by Diegogagan2587 — APPROVED** [review]
# Status Approved! :green_circle:

**Pull request merged**